doc/sphinx-guides/source/api/apps.rst

```diff
@@ -102,9 +102,9 @@ Please note that there is a Java library for Dataverse Software APIs listed in t
 DVUploader
 ~~~~~~~~~~
 
-The open-source DVUploader tool is a stand-alone command-line Java application that uses the Dataverse Software API to upload files to a specified Dataset. Files can be specified by name, or the DVUploader can upload all files in a directory or recursively from a directory tree. The DVUploader can also verify that uploaded files match their local sources by comparing the local and remote fixity checksums. Source code, release 1.0.0- jar file, and documentation are available on GitHub. DVUploader's creation was supported by the Texas Digital Library.
+The open-source DVUploader tool is a stand-alone command-line Java application that uses the Dataverse Software API to upload files to a specified Dataset. Files can be specified by name, or the DVUploader can upload all files in a directory or recursively from a directory tree. The DVUploader can also verify that uploaded files match their local sources by comparing the local and remote fixity checksums. Source code, release 1.0.9- jar file, and documentation are available on GitHub. DVUploader's creation was supported by the Texas Digital Library.
```

Comment:
1.1.0 is out - I updated the wiki. Perhaps just removing the version here would be easier?

Reply:
@qqmyers  Easy to remove the version here in "apps", but there is an example in the User Guide. How should that be handled (i.e., with versions)?
https://dataverse-guide--8852.org.readthedocs.build/en/8852/user/dataset-management.html